spec/features/user_sees_homepage_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+feature 'Visiting Homepage' do
+  context 'as a visitor' do
+    before :each do
+      visit root_path
+    end
+    it 'shows app name, tag line, and synopsis' do
+      expect(page).to have_content("Range Society")
+      expect(page).to have_content("Live Within Reach")
+      expect(page).to have_content("Discover locations and activities within your electric vehicle's range")
+    end
```

Comment:
Looks great! Love the quick start to TDD this app!

Reply:
Gotta love the TDD. We Thanks for the quick start on the testing